tamr_unify_client/models/project/estimated_pair_counts.py

```diff
@@ -0,0 +1,42 @@
+from tamr_unify_client.models.base_resource import BaseResource
+
+
+class EstimatedPairCounts(BaseResource):
+    """Estimated Pair Counts info for Mastering Project"""
+
+    @classmethod
+    def from_json(cls, client, resource_json, api_path=None) -> "EstimatedPairCounts":
+        return super().from_data(client, resource_json, api_path)
+
+    @property
+    def is_up_to_date(self) -> bool:
+        """Whether the associated dataset is up to date.
+
+        :type: bool
+        """
+        return self._data.get("isUpToDate")
+
+    @property
+    def total_estimate(self) -> dict:
+        """Info about when profile info was generated.
+
+        :type: dict
+        """
+        return self._data.get("totalEstimate")
+
+    @property
+    def clause_estimates(self) -> dict:
+        """Info about when profile info was generated.
```

Comment:
Docstring needs to describe structure of results, i.e. what are the keys and what are the values in the returned `dict`